manifests/j/JakobDev/jdMinecraftLauncher/4.1/JakobDev.jdMinecraftLauncher.installer.yaml

```diff
@@ -0,0 +1,19 @@
+PackageIdentifier: JakobDev.jdMinecraftLauncher
+PackageVersion: 4.1
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
+Installers:

```

Comment:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
    - PackageIdentifier: Microsoft.VCRedist.2015+.x86
Installers:
```

Reply:
I will include this in the Build as explained below